README.md

```diff
@@ -125,6 +125,12 @@ function gtst() {
     gcap "✅ TEST: $@"
 }
 ```
+For Linux user who use the graphical program GIMP, use this alternative IMPROVE function:
+```
```

Comment:
I think we don't need to mention that. Just add the snippet with a comment inside the code snippets instead of making separate sections.